src/engine/map.cxx

```diff
@@ -361,7 +360,7 @@ void Map::highlightNode(const Point &isoCoordinates)
   {
     int index = isoCoordinates.x * _columns + isoCoordinates.y;
 
-    if (index >= 0 && index < _mapNodes.size())
+    if (index >= 0 && index < static_cast<int>(_mapNodes.size()))
```

Comment:
I think that changing the type of `index` to `size_t` is better.

Reply:
i agree. done